ci-operator/step-registry/ipi/aws/pre/ipi-aws-pre-chain.yaml

```diff
@@ -3,6 +3,12 @@ chain:
   steps:
   - chain: ipi-conf-aws
   - chain: ipi-install
+  env:
+    - name: COMPUTE_WORKER
```

Comment:
nit: I think this should be `COMPUTE_NODE_TYPE` to distinguish from other compute properties (e.g. zones).